docs/tutorials/python-development-environment.md

```diff
@@ -0,0 +1,121 @@
+
+    nano myproject.git/hooks/post-receive
+
+add these lines:
+
+    #!/bin/sh
+    git --work-tree=/home/anton/myproject --git-dir=/home/anton/myproject.git checkout -f
+    
+and finally do:
+
+    chmod +x myproject.git/hooks/post-receive
+
+## On to PyCharm ##
+
+Now typing code in nano is fine, but it's much nicer to do it inside a real IDE (Integrated Development Environment). There are many, like Visual Studio, Eclipse and PyCharm. My Favourite is PyCharm Community Edition. So go grab a copy of that one and start it on your development machine.
+
+{% include inline-screenshot.html source="/images/osx/PyCharm/welcome.png" caption="The 'Welcome' dialog in PyCharm"%}
+
+What we are going to do now is make a clone of our project on the ev3dev machine to start working on it on the development machine.
+In the Welcome dialog choose: 'Checkout from version control' > 'Git'
+Now type the hostname of the ev3dev machine, followed by a semicolon the projectname. In the other fields choose a nice parent and project directory.
+
+When all goes wel you get a new window with your fresh empty project. If the 'testing' dialog stays on screen for a long time, it might be that your PyCharm master password is needed for your PyCharm password storage. Cancel the checkout, type the master password and try again.
+
+So let's add some code. Right click on the 'myproject' folder in the left column and choose new > python file. Name it `run-motor`. PyCharm will ask if you want to add it to Git. That's nice! Of course we want that.
+
+Now add the following code:
+
+{% highlight python %}
+from ev3dev.auto import OUTPUT_A, Motor 
+import time
+
+m = Motor(OUTPUT_A)
+m.run_forever(duty_cycle_sp = 100)
+time.sleep(1)
+m.stop()
+{% endhighlight %}
+
+When you're done press cmd-k or choose VCS > Commit Changes...
+
+The commit message is box describing the changes you made to your code. That's handy later on when you do a lot of changes. For now type: `first commit`. Then go over the commit button and choose "commit and push" from the dropdown. In the next dialog click make sure to select the "alternative branch checkbox" and click "Push". Voila! Our code has arrived on the ev3dev brick. Let's take a look.
+
+## running the code ##
+
+It's time to let the first motor run. Plug it into port A on your brick and ssh to your ev3dev brick. Maybe you have the terminal still open. 
+Now go
+
+    cd myproject
+    python run_motor.py
+
+Tataaa! The motor is running full power for a second.
+
+## Installing ev3dev-python on the development machine ##
+Back to the development machine. Maybe you noticed a problem: PyCharm puts red curly lines under the ev3dev library. 
+
+![Curly lines]({{ site.github.url }}/images/osx/PyCharm/missing-lib.png =435)
```

Comment:
omit `{{ site.github.url }}`

Also, for screenshots, we have a special template.

{% include screenshot.html source="/images/osx/PyCharm/missing-lib.png" %}